modules/openapi-generator/src/main/resources/JavaJaxRS/spec/pojo.mustache

```diff
@@ -71,12 +93,12 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}} {{#vendorExtens
   {{#vendorExtensions.x-extra-annotation}}{{{vendorExtensions.x-extra-annotation}}}{{/vendorExtensions.x-extra-annotation}}{{#useSwaggerAnnotations}}
   @ApiModelProperty({{#example}}example = "{{{.}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}value = "{{{description}}}"){{/useSwaggerAnnotations}}{{#useMicroProfileOpenAPIAnnotations}}
   @org.eclipse.microprofile.openapi.annotations.media.Schema({{#example}}example = "{{{.}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}description = "{{{description}}}"){{/useMicroProfileOpenAPIAnnotations}}
-  @JsonProperty("{{baseName}}")
+  @JsonProperty({{#required}}required = {{required}}, {{/required}}value = "{{baseName}}")
```

Comment:
Moving the close of the `required` check could eliminate adding `value` in case where it is redundant.
```suggestion
  @JsonProperty({{#required}}required = {{required}}, value = {{/required}}"{{baseName}}")
```

Reply:
Would also significantly shrink the changeset of this PR 😄

Reply:
Thanks for the review!
Doing as you suggested is indeed better, diff is "only" 64 files modified now :smiley: 
(Had to change a bit more than your suggestion to get rid of an unwanted line break)

Reply:
Something is not right, in case #required but no #generateBuilders, we wouldn't get an empty constructor and this could break some existing code.
I updated the PR to always generate an empty constructor unconditionally, so diff is less small :smiling_face_with_tear: